rlclientlib/live_model_impl.cc

```diff
@@ -417,4 +423,35 @@ namespace reinforcement_learning {
     return error_code::success;
   }
 
+  int post_process_rank(ranking_response& response, decision_mode decision_mode) {
+    if (decision_mode == IMITATION_MODE) {
```

Comment:
I prefer a bool here but if its not, the enum should be in a switch and non explicitly handled cases should return error.

Reply:
Changed to switch-case statement.